web/html/src/manager/systems/bootstrap/bootstrap-minions.tsx

```diff
@@ -17,6 +19,81 @@ declare global {
   }
 }
 
+type ErrorDetails = {
+  message: string;
+  standardOutput?: string;
+  standardError?: string;
+  result?: string;
+};
+
+type ErrorDetailsPopUpProps = {
+  error: ErrorDetails | null;
+  onClosePopUp: () => void;
+};
+
+class ErrorDetailsPopUp extends React.Component<ErrorDetailsPopUpProps> {
+  render() {
+    let content, title, buttons;
+
+    if (this.props.error) {
+      title = (
+        <span>
+          <i className="fa fa-list" /> {t("Error Details")}
+        </span>
+      );
+
+      content = (
+        <>
+          <p>{this.props.error.message}</p>
+          {this.props.error.standardOutput && (
+            <div className="form-group">
+              <label className="control-label">Standard Output:</label>
+              <textarea readOnly disabled className={`form-control`} value={this.props.error.standardOutput} rows={5} />
+            </div>
+          )}
+          {this.props.error.standardError && (
+            <div className="form-group">
+              <label className="control-label">Standard Error:</label>
+              <textarea readOnly disabled className={`form-control`} value={this.props.error.standardError} rows={5} />
+            </div>
+          )}
+          {this.props.error.result && (
+            <div className="form-group">
+              <label className="control-label">Result:</label>
+              <textarea readOnly disabled className={`form-control`} value={this.props.error.result} rows={5} />
+            </div>
+          )}
+        </>
+      );
+
+      buttons = (
+        <div>
+          <Button
+            className="btn-default"
+            text={t("Close")}
+            title={t("Close")}
+            icon="fa-close"
+            handler={() => {
+              jQuery("#show-error-details").modal("hide");
```

Comment:
It would be preferable not to use jQuery in any new code. Is there a case the React-based modal logic doesn't cover?

Reply:
No, no specific reason. This is just coming from the other page I copy-pasted for reference :sweat_smile: . I will fix it.

Reply:
I replaced the jQuery based `PopUp` component with the `Dialog`